### PR TITLE
Adds documentation of verification 

### PR DESCRIPTION
**Pull request by dietmarw**
This  is the added documentation of verification as done on @marcelofcastro 's master branch.

### CODE REVIEW

File: OpenIPSL/Electrical/Machines/PSSE/GENROE.mo [hunk 1]
```diff
@@ -163,5 +163,35 @@ equation
...
+<td><p>Contact</p></td>
+<td><p><a href=\"mailto:vanfrl@rpi.edu\">vanfrl@rpi.edu</a></p></td>
+</tr>
+<tr>
+<td><p>Model Verification</p></td>
+<td><p>This model has been verified against PSS/E.</p></td>
+</tr>
+<tr>
+<td><p>Description</p></td>
+<td><p>Round Rotor Generator Model GENROE Induction Generator Model CIMTR1. It is the same as GENROU model except that an exponential function is used as saturation.</p></td>
```
Comment (dietmarw) [inline OpenIPSL/Electrical/Machines/PSSE/GENROE.mo:193]:
```suggestion
<td><p>Round Rotor Generator Model GENROE. It is the same as GENROU model except that an exponential function is used as saturation.</p></td>
```

**Comment by dietmarw** [comment]
Should be fine now.

**Comment by dietmarw** [comment]
All merge conflicts are now resolved and CI is also green. So just waiting for a review :-)

**Review by marcelofcastro — APPROVED** [review]
merge is approved 👍

**Pull request merged**